### DISCUSSION TITLE
Support for curve 448 based gpg keys are missing

### DISCUSSION THREAD

**Discussion by tuxmaster** [community/community#64381] [Code Security]
### Select Topic Area

Bug

### Body

Are there plans to add this?

**Comment by Tolgahan0** [comment]
Response #2

**Reply by vlakoff** [reply]
I really tried to figure out this comment, but I don't understand it…

**Comment by vlakoff** [comment]
For those interested, I've opened a more elaborate feature request in #180422 for Curve448 support.

**Comment by jesus75pittml-tech** [comment]
> 

> ÿwmal99maaa

Au7w1nCx№![...is 

- [ ] ++

,,n&^&,nncx.x @vlakoff
